lib/matplotlib/figure.py

```diff
@@ -1038,6 +1038,38 @@ def fixlist(args):
         key = fixlist(args), fixitems(kwargs.items())
         return key
 
+    def add_artist(self, a, clip=False):
+        """
+        Add any :class:`~matplotlib.artist.Artist` to the figure.
+
+        If the added artist has no transform set to it previously, its
+        transform will be set to ``transFigure``.
+
+        Use ``add_artist`` only in cases you really need to add an artist to
```

Comment:
This has a lot of "only" and "most often".  Note a big reason to do this is if you don't want the artist in the tighbbox of the axes.  Maybe:

Usually artists are added to axes objects using :meth:`matplotlib.axes.Axes.add_artist`, but use this method in the rare cases that adding directly to the figure is necessary.